web/components/quests-or-streak.tsx

```diff
@@ -201,7 +207,7 @@ const QuestRow = (props: {
           </span>
           <Col>
             <span className={clsx('sm:text-xl')}>
-              {title}
+              {link ? <a href={url}>{title}</a> : title}
```

Comment:
We use NextJS's link components. You could do this like:

`<Link href={href}>{title}</Link>`

Reply:
Makes sense, fixed